examples/run_T1CPMG.py

```diff
@@ -32,190 +32,189 @@
     print("***")
-    SpinCore_pp.configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
-    acq_time_ms = SpinCore_pp.configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps) #ms
-    acq_params['acq_time_ms'] = acq_time_ms
-    SpinCore_pp.init_ppg();
+    SpinCore_pp.configureTX(
+        config_dict["adcOffset"],
+        config_dict["carrierFreq_MHz"],
+        tx_phases,
+        config_dict["amplitude"],
+        nPoints,
+    )
+    acq_time_ms = SpinCore_pp.configureRX(
+        config_dict["SW_kHz"],
+        nPoints,
+        config_dict["nScans"],
+        config_dict["nEchoes"],
+        nPhaseSteps,
+    )  # ms
+    SpinCore_pp.init_ppg()
     if phase_cycling:
-        SpinCore_pp.load([
-            ('marker','start',1),
-            ('phase_reset',1),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,0),
-            ('delay',vd_us),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',p90_us,'ph1',r_[0,2]),
-            ('delay',tau_us),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,1),
-            ('delay',deadtime_us),
-            ('delay',pad_start),
-            ('acquire',acq_time_ms),
-            ('delay',pad_end),
-            ('marker','echo_label',(nEchoes-1)), # 1 us delay
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,1),
-            ('delay',deadtime_us),
-            ('delay',pad_start),
-            ('acquire',acq_time_ms),
-            ('delay',pad_end),
-            ('jumpto','echo_label'), # 1 us delay
-            ('delay',repetition_us),
-            ('jumpto','start')
-            ])
+        SpinCore_pp.load(
+            [
+                ("marker", "start", 1),
+                ("phase_reset", 1),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 0),
+                ("delay", vd_us),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", config_dict["p90_us"], "ph1", r_[0, 2]),
+                ("delay", tau_us),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 1),
+                ("delay", config_dict["deadtime_us"]),
+                ("delay", pad_start),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad_end),
+                ("marker", "echo_label", (config_dict["nEchoes"] - 1)),  # 1 us delay
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 1),
+                ("delay", config_dict["deadtime_us"]),
+                ("delay", pad_start),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad_end),
+                ("jumpto", "echo_label"),  # 1 us delay
+                ("delay", config_dict["repetition_us"]),
+                ("jumpto", "start"),
+            ]
```

Comment:
it's own ppg but not used enough to make a separate function to run t1cpmg unless told otherwise